### ISSUE TITLE
Visualize  train/validation set

### ISSUE THREAD

**Issue by KennyTC** [talmolab/sleap#1051]
After training models, I want  to visualize my validation set to check whether my model performed well or not. But I am struggling to find the way.

I  know  that after training models, I have the  following files. 
![image](https://user-images.githubusercontent.com/16069257/202059206-73b5489f-2b0b-4e33-ad12-106ec35ad530.png)

From https://sleap.ai/notebooks/Model_evaluation.html, I know that the meaning of each file. But I was struggling to find the following

1) What is the frames used for validation?  I opened labels_pr.val.slp file, but it contains all the frames I used for training (train, and validation frames),  not only frames  used for validation.

2) I  want to visualize groundtruth  and predicted  points of validation set on the same screen, how can I do this?

Thanks.

**Comment by roomrys** [comment]
Hi @KennyTC,

1. In the training_config.json, you should see a section named "validation_inds". This section lists the indices of the of the labeled frames within your slp that were used for validation during training. The labels_pr.val.slp SHOULD include just the predictions on the validation frames (not the training) - if they are not, then this is a bug. I double checked this on my system and it looks correct to me so it is possible it could be a system specific bug.
> Note: the validation fraction is rounded to include at least one frame in both the test and validation sets.

> Note: the indices referred to here are not the same as the frame index (in a video). The indices refer to the [`LabeledFrame`](https://sleap.ai/api/sleap.instance.html#sleap.instance.LabeledFrame) index in the list of `LabeledFrame`s contained in the project's [`Labels.labeled_frames`](https://sleap.ai/api/sleap.io.dataset.html#sleap.io.dataset.Labels.labeled_frames) attribute and will be stored in the order in which you added the labeled frame.
2. To visualize both the ground truth and predicted points of the validation set, you can open labels_gt.val.slp and File > Merge into project the labels_pr.val.slp.

Let me know if this was helpful. Do you mind double checking that labels_pr.val.slp and labels_pr.train.slp contain different frames?

Thanks,
Liezl

**Comment by roomrys** [comment]
Converting this to a discussion for now. If there is indeed an issue where labels_pr.val.slp includes both the predictions on the validation frames and on training frames, then we can re-open this issue.
